### ISSUE TITLE
[bug]: Flutter web when we reload page on web browser , it will lost back button 

### ISSUE THREAD

**Issue by Kimsoer** [Stacked-Org/stacked#1031]
### Describe the bug

In Flutter web, When we reload page on browser, it will lost back button, and if we use back button, then we add nav             navigationService.pop(); it also will not back to previous screen

### To reproduce

1.click navigate from home screen
2.It will open detail second screen
3.click Reload refresh on browser, it will lost button back, if we add button back with leading, it will not work back to previous event we use pop function back.

**Comment by ferrarafer** [comment]
Hi @Kimsoer , I think this is the default behavior on Flutter, not a Stacked issue. Did you tried the same without Stacked (only Flutter code) ?

When you click on refresh button on the browser, the app is rebuild from scratch so you need to do some extra work to put the app in the previous state.

**Comment by FilledStacks** [comment]
Yes, this would be a new feature to implement in Stacked. It's not implemented by default.

**Comment by Kimsoer** [comment]
Noted with thanks @ferrarafer
